CredentialPostTest/CredentialPostTest/wwwroot/js/site.js

```diff
@@ -35,9 +58,25 @@ function processIntegrationCreatedResult(systems) {
     }
     let sourceSystem = systems[0];
     let targetSystem = systems[1];
-    var message = document.getElementById('message');
+    let message = document.getElementById('message');
     if (message) {
         message.removeAttribute("hidden");
         message.innerHTML = 'Integration between <b>' + sourceSystem + '</b> and <b>' + targetSystem + '</b> was created successfully!';
     }
+}
+
+function processClientAuthenticatedMessage(authMessage){
+    let code = authMessage.authorizationCode;
+    callApi('GET', '/Index?handler=AccessToken&authCode=' + code, function(data){
+        let accToken = data.accessToken;
```

Comment:
Access token in plain text?!

Reply:
We can return encrypted access token to client from our API.

Reply:
Or we can pass the authorizationCode to window so exchange can be done serverside?